cypress/test/dashboard/table/filters.cy.js

```diff
@@ -0,0 +1,56 @@
+import { mount } from 'cypress/react';
+import Filters from '../../../../components/dashboard/table/Filters';
+
+
+describe('Filters Component', () => {
+  it('renders the component', () => {
+    mount(
+      <Filters
+        className="test-class"
+        issues={[]}
+        setSelectedRepo={() => {}}
+        allIssues={[]}
+        setSelectedArea={() => {}}
+        selectedArea=""
+        selectedRepo=""
+      />
```

Comment:
Test for filters shouldn't be tested only for it's existence, it's important that we should test it's functionality too by passing correct attributes inside it.

Reply:
done the required changes in current commit .